applications/DamApplication/custom_strategies/schemes/bossak_displacement_smoothing_scheme.hpp

```diff
@@ -57,6 +57,21 @@ class BossakDisplacementSmoothingScheme : public ResidualBasedBossakDisplacement
         r_model_part.GetProcessInfo()[RAYLEIGH_ALPHA] = mRayleighAlpha;
         r_model_part.GetProcessInfo()[RAYLEIGH_BETA] = mRayleighBeta;
 
+        const int NNodes = static_cast<int>(r_model_part.Nodes().size());
+        ModelPart::NodesContainerType::iterator node_begin = r_model_part.NodesBegin();
+
+        // Initialize INITIAL_STRESS_TENSOR
+        #pragma omp parallel for
+        for(int i = 0; i < NNodes; i++)
```

Comment:
If you are adding new parallel stuff, please use the methods defined in the `parallel_utilities.h`. Note that this are intentionally designed to interface the pragmas aiming at a smoother transition to the upcoming c++ native parallelism.

Reply:
Hey Ruben, I've been trying to find an alternative in the `parallel_utilities.h` file, but I'm not sure about the necessary changes. Could you please show me an example of the method I should use in a file that's already implemented in Kratos?

Reply:
> If you are adding new parallel stuff, please use the methods defined in the `parallel_utilities.h`. Note that this are intentionally designed to interface the pragmas aiming at a smoother transition to the upcoming c++ native parallelism.

Hey Ruben, I've been trying to find an alternative in the `parallel_utilities.h` file, but I'm not sure about the necessary changes. Could you please show me an example of the method I should use in a file that's already implemented in Kratos?

Reply:
Sure. You can see a case similar to yours in here https://github.com/KratosMultiphysics/Kratos/blob/9dbf04851f80f4ef658387a3f86307997349b3df/applications/FluidDynamicsApplication/custom_strategies/strategies/compressible_navier_stokes_explicit_solving_strategy.h#L404-L410

In any case, in your case you can make it easier by doing
```c++
block_for_each(r_model_part.Nodes(), [](Node& rNode){
    auto& r_initial_stress = rNode.FastGetSolutionStepValue(INITIAL_STRESS_TENSOR);
    if(r_initial_stress.size1() != 3)
        r_initial_stress.resize(3,3,false);
    noalias(r_initial_stress) = ZeroMatrix(3,3);
});
```